qiita_pet/support_files/doc/source/gettingstartedguide/index.rst

```diff
@@ -272,34 +259,6 @@ prep information file to be added to the system:
 |                   |                               | match the sample_name in the sample template.                          |
 +-------------------+-------------------------------+------------------------------------------------------------------------+
 
-Required fields for EBI submission
```

Comment:
@antgonza Is it okay that the section 'Required fields for EBI submission' has been removed?

Reply:
Thank you for checking. Yes, it has been merged into "Required fields for EBI submission and to make a study public"